res/css/views/messages/_DecryptionFailureBody.pcss

```diff
@@ -10,3 +10,24 @@ Please see LICENSE files in the repository root for full details.
     color: $secondary-content;
     font-style: italic;
 }
+
+/* Formatting for the "Verified identity has changed" error */
+.mx_DecryptionFailureVerifiedIdentityChanged > span {
+    /* Show it in red */
+    color: $e2e-warning-color;
+    /* TODO: background colour? */
+
+    /* With a red border */
+    border: 1px solid $e2e-warning-color;
+    border-radius: $font-16px;
+
+    /* Some space inside the border */
+    padding: 4px 12px 4px 8px;
```

Comment:
Can we use compound [spacing token](https://compound.element.io/?path=/docs/tokens-spacing--docs)

Reply:
I can't really figure out how to use those tokens in css.  They have names like "cpdSpace1X", but we don't seem to use that anywhere currently?

Reply:
Yes we are with this naming: `var(--cpd-space-1x)`

Reply:
updated. Thanks!